model/atmosphere/dycore/tests/dycore_tests/test_velocity_advection.py

```diff
@@ -232,29 +243,21 @@ def test_velocity_predictor_step(
     if not vn_only:
         assert helpers.dallclose(
             icon_result_z_v_grad_w[start_edge_lateral_boundary_6:, :],
-            velocity_advection.z_v_grad_w.asnumpy()[start_edge_lateral_boundary_6:, :],
+            velocity_advection._khalf_horizontal_advection_of_w_at_edge.asnumpy()[
+                start_edge_lateral_boundary_6:, :
+            ],
             atol=1.0e-16,
         )
 
     # stencil 08
     start_cell_nudging = icon_grid.start_index(h_grid.domain(dims.CellDim)(h_grid.Zone.NUDGING))
     assert helpers.dallclose(
         savepoint_velocity_exit.z_ekinh().asnumpy()[start_cell_nudging:, :],
-        velocity_advection.z_ekinh.asnumpy()[start_cell_nudging:, :],
-    )
-    # stencil 09
-    assert helpers.dallclose(
```

Comment:
has this field been removed?

Reply:
This field is now calculated and used in a fused stencil, hence there is no need to return it